cd/application-deployment/dev/dev.env

```diff
@@ -27,6 +27,7 @@ PII_ENABLED=False
 PINPOINT_SMS_VOICE_V2=False
 PLATFORM_STATS_ENABLED=True
 REDIS_ENABLED=True
+REVISED_TEMPLATE_RENDERING=True
```

Comment:
We don't want this feature turned on yet. It will break test sends in the lower envs. Portal team has zero time to do anything about that right now. We can revisit when we start looking into template caching.

Reply:
I changed it to False (because QA already tested it).